src/generator/docker-compose-file-generator.ts

```diff
@@ -45,7 +170,60 @@ export class DockerComposeFileGenerator extends AbstractFileGenerator {
         );
     }
 
-    generateDevelopmentServiceDockerComposeFile (service: Service, builderVersion: string | undefined) {
+    /**
+     * Generates exclusion filtered Docker Compose files in the `exclusion-filtered-compose-files` directory and updates the main docker-compose.yaml file `includes` property as a list of the generated exclusion filtered docker-compose files.
+     * create exlusion filtered docker-compose for non-development mode services.
+     * update development-mode services docker-compose file by removing excluded services from its depend_on property.
+     * update main docker-compose file `include` and `depends_on` properties appropriately.
+     * @param services - List of all services with live updates.
+     * @param excluded - List of excluded service names.
+     */
+    generateExclusionDockerComposeFiles (services: ServiceWithLiveUpdate[], excluded: string[] | undefined) {
+        const exclusionDirPath = join(this.path, "exclusion-filtered-compose-files");
```

Comment:
Ensure this path is added to the `.gitignore` of `docker-chs-development` I can see what will happen otherwise...

Reply:
I will create PR into docker-chs-development once this is merged.